README.md

```diff
@@ -40,6 +40,14 @@ From source:
     gem build brakeman.gemspec
     gem install brakeman*.gem
 
+## For Ruby 1.8.7 Users
+
+Slim v3.0.0 dropped support for Ruby 1.8.7. If you're using `brakeman` on Ruby 1.8.7 you will have to pre-install a version of `slim` compatible with Ruby 1.8.7:
+
+    $ gem install slim --version '~> 2.0.0'
+
+If you're on a more recent Ruby this step is _not_ required.
```

Comment:
This is a little misleading...you need to install slim manually if you want to process slim templates, either way.